README.md

```diff
@@ -68,6 +68,36 @@ Once you have your dependencies configured, you can import packages using your i
 import "@openzeppelin/token/ERC721/ERC721.sol";
 ```
 
+### Library Linking
+
+To link known pre-deployed libraries, add them to your `ape-config.yaml` file:
+
+```yaml
+solidity:
+  libraries:
+    subfolder/MySolidityFile.sol:
+      MyLibrary: "0x3387FE7316B9418152F338E5A90D3a2C2888a1FF"
+```
+
+**WARNING**: If your library address changes across networks, you have to edit your config to use the new address and force re-compile.
```

Comment:
Should this be per-network? e.g.:
```yaml
solidity:
  libraries:
    subfolder/MySolidityFile.sol:
      MyLibrary:
        ethereum:
          mainnet: "0x3387FE7316B9418152F338E5A90D3a2C2888a1FF"
```

Can just attempt to fetch the provider, and if it doesn't exist then just raise that you can't link against code unless connected to a network? I suppose this makes `ape compile` fail require a network arg now....

Reply:
Yes, haha, you are going down the same trail I did.
There is not an easy to know which network the user wants to use when linking addresses at compile time.

It does make sense to have this be per network though. I am just not sure how.

We could exclude the config-related changes for now since only the adhoc approach is more critical at this time.